frontend/src/pages/Notifications/NotificationsSnackbar.js

```diff
@@ -20,6 +22,9 @@ const styles = theme => ({
   error: {
     backgroundColor: theme.palette.error.main
   },
+  warning: {
+    backgroundColor: theme.palette.warning
```

Comment:
we should add a lighter/darker warning color to the warning palette since the warning snackbar is difficult to read

Reply:
I set the color for the warning darker, since the warning color is only used for this warning I think it's not necessary to add a lighter and darker